lib/ui/widgets/patcherView/app_selector_card.dart

```diff
@@ -35,25 +35,31 @@ class AppSelectorCard extends StatelessWidget {
           if (locator<PatcherViewModel>().selectedApp == null)
             I18nText('appSelectorCard.widgetSubtitle')
           else
-            Row(
-              children: <Widget>[
-                SizedBox(
-                  height: 18.0,
-                  child: ClipOval(
-                    child: Image.memory(
-                      locator<PatcherViewModel>().selectedApp == null
-                          ? Uint8List(0)
-                          : locator<PatcherViewModel>().selectedApp!.icon,
-                      fit: BoxFit.cover,
+            LayoutBuilder(
+              builder: (BuildContext context, BoxConstraints constraints) {
+                return Row(
+                  children: <Widget>[
```

Comment:
Remove the `LayoutBuilder` as it is not required now.

Reply:
@TheAabedKhan Just did that, anything else needed?